app/config/variables.php

```diff
@@ -920,7 +920,17 @@
                 'required' => false,
                 'question' => '',
                 'filter' => ''
+            ],
+            [
+            'name' => '_APP_MAINTENANCE_RETENTION_SCHEDULES',
+            'description' => 'Deleted functions schedule cleanup interval (in seconds) ',
+            'introduction' => '1.2.0',
+            'default' => '86400',
+            'required' => false,
+            'question' => '',
+            'filter' => ''
```

Comment:
Just a formatting thing. Lets add this variable near the other maintenance variables